pkg/operator/certrotation/signer_test.go

```diff
@@ -50,13 +51,24 @@ func TestEnsureSigningCertKeyPair(t *testing.T) {
 				if len(actual.Data["tls.crt"]) == 0 || len(actual.Data["tls.key"]) == 0 {
 					t.Error(actual.Data)
 				}
+				if len(actual.Annotations) == 0 {
+					t.Errorf("expected certificates to be annotated")
+				}
+				ownershipValue, found := actual.Annotations[annotations.OpenShiftComponent]
```

Comment:
I prefer to avoid sharing values like this between tests and implementation, since it creates the opportunity to accidentally make a breaking change without seeing a test failure.

Feel free to ignore if you disagree or are comfortable with it in this instance because it is declared in openshift/api.

Reply:
>it creates the opportunity to accidentally make a breaking change without seeing a test failure

We explicitly set `AdditionalAnnotations` in https://github.com/openshift/library-go/pull/1681/files#diff-d7a4e30c6635930d8721bf060fe2803587326298e0eace9bada256f1b7377459R228-R230

Reply:
That link didn't work for me. In this case I'm specifically referring to a change to the constant `OpenShiftComponent`, which would be a breaking change but would not cause this test to fail.

Reply:
Ah, hmm, `JiraComponent` and `OpenshiftComponent` are linked in https://github.com/openshift/library-go/blob/master/pkg/operator/certrotation/annotations.go#L27-L30, not sure how to make sure unit tests don't depend on it though